### DISCUSSION TITLE
AXN HD | (México)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#151] [Show and tell]
### Channel Name

AXN HD

### Channel ID

AXNMexico.mx

### Stream URL

https://edge2-ccast-sl.cvattv.com.ar/live/c7eds/AXNHD/SA_SAGEMCOM/AXNHD.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
<img width="752" alt="image" src="https://user-images.githubusercontent.com/7253922/158995314-db4d07e6-837f-41b1-a784-06287e3611d0.png">

**Comment by R4IN2007** [comment]
https://edge2.prepro.cvattv.com.ar/live/vod/AXNHD/SA_Live_WebVTT_FTA/AXNHD.m3u8

**Comment by BellezaEmporium** [comment]
> https://edge2.prepro.cvattv.com.ar/live/vod/AXNHD/SA_Live_WebVTT_FTA/AXNHD.m3u8

503 for this one on my side.

**Comment by freearhey** [comment]
> 503 for this one on my side.

Now I have an error too.
